### PR TITLE
App/homeassistant-1

### PR DESCRIPTION
**Pull request by falkheiland**
- adds new app homeassistant-1 to add the possibility to expose the app
- deprecates the app homeassistant
- migration guides on both apps

## Summary by CodeRabbit

- **Documentation**
	- Enhanced Home Assistant guide with detailed migration steps for making the app exposable, including directory renaming, configuration modifications, environment variable updates, and app startup instructions.

### CODE REVIEW

File: apps/homeassistant-1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/data/config:/config
+    networks:
+      - tipi_main_network
+    labels:
+      # Main
+      traefik.enable: true
+      traefik.http.middlewares.homeassistant-web-redirect.redirectscheme.scheme: https
+      traefik.http.services.homeassistant.loadbalancer.server.port: 8123
```
Comment (meienberger) [inline apps/homeassistant-1/docker-compose.yml:21]: Shouldn't be 8209?

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/homeassistant-1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+services:
+  homeassistant-1:
+    image: ghcr.io/home-assistant/home-assistant:2024.2.1
+    container_name: homeassistant-1
+    environment:
+      - TZ=${TZ}
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:8209
```
Comment (meienberger) [inline apps/homeassistant-1/docker-compose.yml:11]: This doesn't work in my tests, the internal port 8209 leads to nothing

File: apps/homeassistant-1/docker-compose.yml (hunk 1 above)
Comment (falkheiland) [reply apps/homeassistant-1/docker-compose.yml]: you are correct. fixed.

Comment (falkheiland) [reply apps/homeassistant-1/docker-compose.yml:21]: post changes revoked. jsut the APP_PORT settings is enough. works now

**Pull request merged**